code/modules/power/engines/singularity/singularity.dm

```diff
@@ -257,8 +262,10 @@
 			grav_pull = 15
 			consume_range = 5
 			dissipate = FALSE
+	if(current_size >= STAGE_FIVE)
+		notify_dead()
```

Comment:
Can you document this change in the PR desc?

Reply:
done